solution/ui/regulations/utilities/api.js

```diff
@@ -210,14 +210,6 @@ function fetchJson({
 
 // ---------- helper functions ---------------
 
-function httpApiMock(verb, urlPath, { data, params, response } = {}) {
-    console.log(`Performing an HTTP ${verb} to ${config.apiPath}/${urlPath}`);
-    data && console.log("DATA: ", data);
-    params && console.log("PARAMS: ", params);
-    response && console.log("RESPONSE: ", response);
-    return response;
-}
-
```

Comment:
This isn't used in our app, so I'm removing it here as part of some general housecleaning.